scripts/hudson/quickstart.sh

```diff
@@ -140,15 +140,19 @@ function configure_wildfly {
   cd $WORKSPACE
   wget --user=guest --password=guest -nv https://ci.wildfly.org/httpAuth/repository/downloadAll/WF_Nightly/.lastSuccessful/artifacts.zip
   unzip -q artifacts.zip
-  export WILDFLY_DIST_ZIP=$(ls wildfly-*Final-SNAPSHOT.zip | head -n 1)
-  [ "x$WILDFLY_DIST_ZIP" = "x" ] && echo "Cannot find WildFly distribution zip file" && return 1
-  unzip -q $WILDFLY_DIST_ZIP
-  export JBOSS_HOME=`pwd`/${WILDFLY_DIST_ZIP%.zip}
+  # the artifacts.zip may be wrapping several zip files: artifacts.zip -> wildfly-latest-SNAPSHOT.zip -> wildfly-###-SNAPSHOT.zip
+  local wildflyLatestZipWrapper=$(ls wildfly-latest-*.zip | head -n 1)
+  unzip -q $wildflyLatestZipWrapper
```

Comment:
Remark: we should go through all of our scripts and make sure we are checking return codes (for example this line could fail if there is insufficient space).

Reply:
+1, I added the check